### ISSUE TITLE
Different colorscale when chart is exported

### ISSUE THREAD

**Issue by vijaydewan** [man-group/dtale#322]
Hi, I noticed that when you export a chart, the colorscale is different.  Is there a way to keep the same colorscale in the exported chart?

**Comment by aschonfeld** [comment]
@vijaydewan good catch! I’ll address this for the next release and update you when its out. Apologies for the inconvenience

**Comment by aschonfeld** [comment]
@vijaydewan I fixed this in v1.25.0, I also added the colorscale option to surface charts

**Comment by vijaydewan** [comment]
Thanks!  The colorscale for the simple scatter still seems a bit different (see Dtale - Screen Colors.png vs. Dtale - Export Colors.png, attached).

I also noticed that the trendline has issues when you have a datetime in the x axis.  I did some research and think this may be an issue with plotly itself.  I plugged in a fix in charts.py to convert the datetime to a float when there is a trendline (in the for loop in _build data() in _build_final_scatter(y_val) in charts.py) but I'm sure there's a more elegant solution:

                if trendline:
                    if pd.DataFrame(d["x"])[0].dtype == 'object':
                        d["x"] = pd.to_datetime(pd.DataFrame(d["x"])[0]).dt.strftime('%Y.%m%d').astype(float).values
                    if pd.DataFrame(d[y_val])[0].dtype == 'object':
                        d[y_val] = pd.to_datetime(pd.DataFrame(d[y_val])[0]).dt.strftime('%Y.%m%d').astype(float).values

![Dtale - Screen Colors](https://user-images.githubusercontent.com/71048263/100756677-7765e700-33bb-11eb-999f-ef26a6c12215.png)
![Dtale - Export Colors](https://user-images.githubusercontent.com/71048263/100756681-79c84100-33bb-11eb-9002-00fda2bec16e.png)
